components/button/index.js

```diff
@@ -26,11 +26,24 @@ class Button extends Component {
 	}
 
 	render() {
-		const { href, target, isPrimary, isLarge, isToggled, className, disabled, ...additionalProps } = this.props;
+		const {
+			href,
+			target,
+			isPrimary,
+			isSecondary,
+			isLarge,
+			isSmall,
+			isToggled,
+			className,
+			disabled,
+			...additionalProps, // eslint-disable-line comma-dangle
```

Comment:
Found quite a bit of prior discussion, which consensus seeming as though the specification does _not_ allow a trailing comma in these cases:

- https://github.com/eslint/eslint/issues/7463
- https://github.com/eslint/eslint/issues/7422
- https://github.com/eslint/eslint/issues/7297
- https://github.com/eslint/eslint/pull/7298
- https://github.com/eslint/eslint/commit/52da71e9bfcdb7f13c6077531b992d059af7391a
- https://github.com/eslint/eslint/issues/7370#issuecomment-254107531